src/Utils/api.js

```diff
@@ -55,3 +55,25 @@ export const postComment = (article_id, newComment) => {
       console.log(error);
     });
 };
+
+export const getTopics = () => {
+  return api
+    .get("/topics")
+    .then((res) => {
+      return res.data.topics;
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
+
+export const getArticlesByTopic = (topicSlug) => {
```

Comment:
Instead of this function the getArticles one could be reused with 'axios params'. (See NC notes/Axios/Axios Params)